index.md

```diff
@@ -1,113 +1,253 @@
 
 <div class="intro-text">
-  The <a href="https://github.com/ethereumjs/">EthereumJS</a> community builds Javascript tools implementing core <a href="https://ethereum.org">Ethereum</a>
-  technologies, protocols and APIs for helping developers to interact with the Ethereum network and build
-  their own applications.
+  <a href="https://github.com/ethereumjs/">EthereumJS</a> is an implementation of the core <a href="https://ethereum.org">Ethereum</a> Execution Layer (EL) Protocol stack in TypeScript.
 </div>
 
+<div class="intro-text">
+  <a href="https://github.com/ethereumjs/ethereumjs-monorepo">https://github.com/ethereumjs/ethereumjs-monorepo</a> 
+  &nbsp;{% include repository_small.html name="ethereumjs-monorepo" %}
+</div> 
+
 <h1>Projects</h1>
 
 <div class="repo-group">
-  <h3><i class="fa fa-cogs"></i> VIRTUAL MACHINE</h3>
-  <p>Implementation of the Ethereum virtual machine supporting the latest fork rules.</p>
-  {% include repository.html name="ethereumjs-vm" %}
+  <h3><i class="fa fa-cogs"></i> VIRTUAL MACHINE (EVM)</h3>
+  <p>Implementation of the Ethereum Virtual Machine in TypeScript supporting
+  all hardforks and allowing for easy integration of the EVM in browser applications.</p>
 </div>
 
 <div class="repo-group">
-  <h3><i class="fa fa-sitemap"></i> MERKLE TREE / RLP</h3>
-  <p>Implementation of the core Ethereum data structure.</p>
-  {% include repository.html name="merkle-patricia-tree" %}
-  {% include repository.html name="rlp" %}
+  <div class="repo-box">
+    <div class="repo-header">
+      <div class="repo-header-right">
+        <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/vm" alt="GitHub URL">
+          <i class="fa fa-github"></i>
+        </a>
+      </div>
+      @ethereumjs/vm
+    </div>
+    <div class="repo-description">
+      <ul>
+        <li>All hardforks until Shanghai</li>
+        <li>Preliminary Cancun support (incl. EIP-4844)</li>
+        <li>Extended tracing and debug support</li>
```

Comment:
Can we say "basic" instead of extended here?  Geth has extended tracing ability.  We have the bare minimum.

Reply:
You're so right. 😂